cogs/weightcog.py

```diff
@@ -182,7 +186,6 @@ async def weight(
             csv_writer = csv.writer(csvfile)
             csv_writer.writerow(header_row)  # Write header row
             csv_writer.writerows(entries)
-
         await ctx.send(f"Weight recorded for {date} ({user.display_name}): {weight} kg.")
```

Comment:
You're changing a lot of the newlines. Don't do this.

Reply:
If for example someone changes code in this same file you'll get merge conflicts

Reply:
It happened during formatting `black .`, i dont know how, will keep this in mind for future.